src/Tools/dotnet-counters/Program.cs

```diff
@@ -145,6 +155,19 @@ private static Task<int> Main(string[] args)
                 .AddCommand(ProcessStatusCommandHandler.ProcessStatusCommand("Lists the dotnet processes that can be monitored"))
```

Comment:
Thanks, I added the --counters argument and made it work only when we're attaching at startup so that we're backwards compatible.

i.e. `dotnet-counters collect --counters System.Runtime,Microsoft.AspNetCore.Hosting -- myapp.exe` will work now. 

The old syntax `dotnet-counters collect System.Runtime Microsoft.AspNetCore.Hosting` also still works.

Reply:
> I added the --counters argument and made it work only when we're attaching at startup

I'd suggest making the --counters argument work all the time and document it as the intended way people should use the tool. Specifying counters as positional arguments becomes the thing that we allow when --process-id is specified to preserve back-compat for people running it via the previous command-line syntax.

Reply:
Done :) Example output below:

```
E:\diagnostics>dotnet-counters collect --process-id 83380 System.Runtime
[Warning] counter.csv already exists. This file will be overwritten.
Starting a counter session. Press Q to quit.
File saved to counter.csv

E:\diagnostics>dotnet-counters collect --process-id 83380 --counters System.Runtime
[Warning] counter.csv already exists. This file will be overwritten.
Starting a counter session. Press Q to quit.
File saved to counter.csv

E:\diagnostics>dotnet-counters collect --counters System.Runtime -- \temp\testapp\bin\Debug\net5.0\testapp.exe
[Warning] counter.csv already exists. This file will be overwritten.
Starting a counter session. Press Q to quit.
File saved to counter.csv
```